test/pixel_format_converter/converter_test.exs

```diff
@@ -0,0 +1,67 @@
+defmodule Membrane.FFmpeg.SWScale.PixelFormatConverter.Test do
+  use ExUnit.Case
+
+  import Membrane.Testing.Assertions
+
+  alias Membrane.RawVideo
+  alias Membrane.FFmpeg.SWScale.PixelFormatConverter
+  alias Membrane.Buffer
+  alias Membrane.Testing.Pipeline
+
+  # All black 4x4 RGBA image
+  @input [<<0x00, 0x00, 0x00, 0xFF>>] |> Stream.cycle() |> Enum.take(4 * 4) |> Enum.join()
+  # All black 4x4 RGB24 image
+  @output [<<0x00, 0x00, 0x00>>] |> Stream.cycle() |> Enum.take(4 * 4) |> Enum.join()
+
+  @input_caps %RawVideo{
+    framerate: {30, 1},
+    aligned: true,
+    width: 4,
+    height: 4,
+    pixel_format: :RGBA
+  }
+
+  test "PixelFormatConverter can convert a single frame from RGBA to RBG" do
+    assert {:ok, state} = PixelFormatConverter.handle_init(%PixelFormatConverter{format: :RGB})
+
+    assert {{:ok, caps: {:output, %RawVideo{pixel_format: :RGB}}}, state} =
+             PixelFormatConverter.handle_caps(:input, @input_caps, nil, state)
+
+    assert {{:ok, buffer: {:output, %Buffer{payload: output}}}, _state} =
+             PixelFormatConverter.handle_process(:input, %Buffer{payload: @input}, nil, state)
+
+    assert byte_size(output) == byte_size(@output)
+    assert output == @output
+  end
+
+  test "integration test" do
+    output_path = "/tmp/output-rgb-10-400x400.raw"
+    reference_path = "test/fixtures/output-rgb-10-400x400.raw"
+
+    opts = %Pipeline.Options{
+      elements: [
+        source: %Membrane.File.Source{location: "test/fixtures/input-10-400x400.raw"},
```

Comment:
Don't use relative paths, they will break when `mix test` is executed from different dir than the repo root
```suggestion
    reference_path = "../fixtures/output-rgb-10-400x400.raw" |> Path.expand(__DIR__)
    input_path = "../fixtures/input-10-400x400.raw" |> Path.expand(__DIR__)

    opts = %Pipeline.Options{
      elements: [
        source: %Membrane.File.Source{location: input_path},
```